app/features/project/views/new_project_form.erb

```diff
@@ -100,6 +141,32 @@
       <select name="selected_lane", id="selected_lane" onchange="check_lane()">
       </select>
       <br />
+      <h5 id="trigger">Select trigger</h5>
+      <div id="trigger_selector">
+        <select name="selected_trigger", id="selected_trigger", onchange="check_trigger(this)">
+          <option disabled selected value>Select a trigger</option>
+          <option value="commit">Commit</option>
```

Comment:
We could populate these values on the server-side instead of hard coding, but I don't think you should prioritize that. Just in the future, know that's an option.